src/state/__snapshots__/experiments.test.js.snap

```diff
@@ -1,37 +1,200 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`experiments/reducer CREATE 1`] = `
+Object {
+  "createInProgress": true,
+  "error": null,
+  "list": Array [],
+  "loading": true,
+  "updateInProgress": false,
+  "uploadInProgress": false,
+  "uploadResult": "ca/experiments/UPLOAD_RES_NONE",
+}
+`;
+
+exports[`experiments/reducer CREATE_ERROR 1`] = `
+Object {
+  "createInProgress": false,
+  "error": "test error",
+  "initialState": Object {
```

Comment:
it's incorrect.

Reply:
Yeah, I missed the spread operator while c&p and of course the tests were not failing :smile: